autopilot/scanner.go

```diff
@@ -288,14 +297,8 @@ func (s *scanner) launchScanWorkers(ctx context.Context, w scanWorker, reqs chan
 	return respChan
 }
 
-func (s *scanner) isScanning() bool {
-	s.mu.Lock()
-	defer s.mu.Unlock()
-	return s.scanning
-}
-
-func (s *scanner) isScanRequired() bool {
-	return s.scanningLastStart.IsZero() || time.Since(s.scanningLastStart) > s.scanMinInterval/20 // check 20 times per minInterval, so every 30 minutes
+func (s *scanner) isScanRequired(force bool) bool {
+	return force || s.scanningLastStart.IsZero() || time.Since(s.scanningLastStart) > s.scanMinInterval/20 // check 20 times per minInterval, so every 30 minutes
```

Comment:
I don't think it makes sense to move `force` in here. It overrules `isScanRequired` and doesn't need to be in here.